html/pfappserver/root/static.alt/src/globals/configuration/pfConfigurationProvisionings.js

```diff
@@ -715,6 +806,20 @@ export const pfConfigurationProvisioningFields = {
         }
       ]
     }
+  },
+  domains: ({ options: { meta = {} } } = {}) => {
+    return {
+      label: i18n.t('Authorized domains'),
+      text: i18n.t('Comma-separated list of domains that will be resolve with the correct IP addresses.'),
```

Comment:
Reword:
A comma-separated list of domains that will be resolved with the correct IP addresses.